app/assets/stylesheets/buttons.scss

```diff
@@ -0,0 +1,22 @@
+@import 'variables';
+@import '_mixins';
+
+.cta {
+  color: $black;
+  background: $green;
+  font-family: $helvetica-condensed;
+  -webkit-appearance: none;
+  font-stretch: condensed;
+  font-weight: bold;
+}
+
+.cta-button {
```

Comment:
The name of this class isn't explanatory/semantic, what would cta be in this case?

Reply:
No idea. I've created this file to start isolating all the button classes that were created elsewhere to be in one single file. Then we will consolidate these and have unified buttons everywhere, probably using some class name like `btn` or `button` with different [variants](https://www.figma.com/file/4FOzCCKLZaLBz6X5TZCVNF/Design-System?node-id=100%3A143).

Reply:
Would `primary` and `secondary` be clearer than `cta`?

Reply:
I definitely think so!